pkgs/by-name/ba/bashrun2/package.nix

```diff
@@ -0,0 +1,90 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  makeWrapper,
+  xorg,
+  ncurses,
+  coreutils,
+  bashInteractive,
+  gnused,
+  gnugrep,
+  glibc,
+  xterm,
+  util-linux,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bashrun2";
+  version = "0.2.6";
+
+  src = fetchFromGitHub {
+    owner = "hbekel";
+    repo = "bashrun2";
+    rev = "v${version}";
```

Comment:
`tag = "v${version}"` is a (slightly) safer option in case there is ever a branch with name matching the tag